dockerfiles/docker-compose.yml

```
@@ -35,17 +35,15 @@ services:
   wrangler:
     build:
       context: ${PWD}
```

We need to pass the correct context here to be able to use the `COPY` commands in the Dockerfile:

```suggestion
      context: ${PWD}/common/packages/addons-inject/
```